astropy/cosmology/io/_parameter.py

```diff
@@ -0,0 +1,74 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
```

Comment:
For consistency with other parts of astropy, my tendency would be to call the module something like `utils` or `helpers` and make all functions public (but obviously not imported in `__init__.py`); as is, private functions are generally not imported elsewhere.

Reply:
SGTM. utils it is.

Reply:
Done.